code/modules/power/apc.dm

```diff
@@ -570,9 +571,10 @@
 				(istype(W, /obj/item/device/multitool) || \
 				istype(W, /obj/item/weapon/wirecutters) || istype(W, /obj/item/device/assembly/signaler)))
 				return src.attack_hand(user)
-			user.visible_message("<span class='warning'>The [src.name] has been hit with the [W.name] by [user.name]!</span>", \
+			/*user.visible_message("<span class='warning'>The [src.name] has been hit with the [W.name] by [user.name]!</span>", \
 				"<span class='warning'>You hit the [src.name] with your [W.name]!</span>", \
-				"You hear bang")
+				"You hear bang")*/
+			..(W,user) //Sanity

```

Comment:
Not really.
You actually cause a bigger issue if the arg order ever changes, and ..() means just as much as with the args unless you change them.